### DISCUSSION TITLE
Best Practices Snapshot VM with Kubernetes + Rook-ceph

### DISCUSSION THREAD

**Discussion by osaffer** [rook/rook#10245] [General]
Hello,

What are the best practices concerning the snapshot of VM with Kubernetes and rook-ceph ?

We use tivoli to snapshot the VM, but we are sometimes faced with a freeze of the VM, or sometimes. they are not frozen, but we encounter some error especially with pods with Database.

May you share your experiences ?

**Comment by travisn** [comment]
Other than a snapshot of the VM, you can also [snapshot the Rook volumes](https://rook.github.io/docs/rook/latest/ceph-csi-snapshot.html).
